ludwig/utils/strings_utils.py

```diff
@@ -36,6 +36,8 @@
 COMMA_REGEX = re.compile(r'\s*,\s*')
 UNDERSCORE_REGEX = re.compile(r'\s*_\s*')
 
+BOOL_TRUE_STRS = {'yes', 'y', 'true', 't', '1'}
```

Comment:
as we are not lowercasing anywhere (I believe) maybe we can add also "Yes", "YES", "Y", "TRUE", "True", "T"

Reply:
The lowercasing occurs in the `str2bool` function below.